model/job.go

```diff
@@ -30,6 +30,17 @@ const (
 	JOB_STATUS_WARNING          = "warning"
 )
 
+var FILE_INFO = map[string]map[string]string{
```

Comment:
`var fileInfo = ..`. There is no need to export this IMO.

Reply:
Moved it inside the function.